src/back/spv/block.rs

```diff
@@ -37,6 +37,28 @@ enum ExpressionPointer {
     },
 }
 
+/// The termination statement to be added to the end of the block
+pub enum BlockExit {
+    /// Generates an OpReturn (void return)
+    Return,
+    /// Generates an OpBranch to the specified block
+    Branch {
+        /// The branch target block
+        block_id: Word,
+    },
+    /// Translates a loop `break if` into an `OpBranchConditional` to the
+    /// merge block if true (the merge block is passed trough [`LoopContext::break_id`]
+    /// or else to the loop header (passed trough [`preamble_id`])
+    ///
+    /// [`preamble_id`]: Self::BreakIf::preamble_id
```

Comment:
Whoa, I didn't know you could use `Self` like that.

Reply:
Item paths in doc comments have the same scope as the item were they are defined, it's very useful.

Shame there's no shorthand for addressing the variant the doc comment is on :(